[DOCS] Fix typo (#75696)
elastic/elasticsearch: docs/reference/search/request/search-after.asciidoc

matches tiebreaker's provided value. Therefore if a document has a tiebreaker value of
`"654323"` and you `search_after` for `"654"` it would still match that document
and return results found after it.

> <<doc-values,doc value>> is disabled on this field so sorting on it requires

Reviewer: ```suggestion
<<doc-values,Doc values>> are disabled on this field so sorting on it requires
```
